source/kernel/version.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018-2021 The Kubernetes Authors.
```

Comment:
2018?

Reply:
Yeah, I used the old year as it's basically just moving existing code to this file. Makes sense?